### ISSUE TITLE
Cannot make install with latest stable version of OCaml (4.11.1)

### ISSUE THREAD

**Issue by Astavie** [mukul-rathi/bolt#142]
It gives this as output on make install:
```
eval 
opam update

<><> Updating package repositories ><><><><><><><><><><><><><><><><><><><><><><>
[default] no changes from https://opam.ocaml.org/
opam install --yes . --deps-only
The following dependencies couldn't be met:
  - bolt → ppx_deriving_protobuf >= 2.7 → ocaml < 4.11
      base of this switch (use `--unlock-base' to force)

No solution found, exiting
Makefile:13: recipe for target 'install' failed
make: *** [install] Error 20
```
I'm currently downgrading my ocaml version, seeing which one works.

**Comment by Astavie** [comment]
OCaml version 4.09.1 works

**Comment by mukul-rathi** [comment]
Hey, that's because of the upstream dependency `ppx_deriving_protobuf`. As far as I can tell, they haven't released a new version to support OCaml 4.11 https://github.com/ocaml-ppx/ppx_deriving_protobuf/releases
